set.mm

```diff
@@ -426627,6 +426616,30 @@ This law is thought to have originated with Aristotle (_Metaphysics_,
     BCDEZFZCAGHUNAIFAJUMIABKFZCKFZLMUMINUPUOUPCCGHZCOFUQMPCQRUPCSFUQCTCUAUBUCUD
     BCKDKKUEKUFDUGUHUIRUJUKUL $.
 
+  ${
+    $( 9 + 10 is not equal to 21.  This disproves a popular meme which asserts
+       that 9 + 10 does equal 21.  See
+       https://www.quora.com/Can-someone-try-to-prove-to-me-that-9+10-21 for
+       attempts to prove that 9 + 10 = 21, and see https://tinyurl.com/9p10e21
+       for the history of the 9 + 10 = 21 meme.  (Contributed by BTernaryTau,
+       25-Aug-2023.) $)
+    9p10ne21 $p |- ( 9 + ; 1 0 ) =/= ; 2 1 $=
+      ( c9 c1 cc0 cdc caddc co 10nn0 nn0cni 9cn dec10p addcomli 1nn0 9nn0 deccl
+      c2 nn0rei 2nn0 9lt10 1lt2 decltc ltneii eqnetri ) AZBZCDZEFUDUCDZOZUDDZUE
+      UCUFUEGHIUCJKUFUHUFUDUCLZMZNPUDUGUCUDUIQUJUIRSTUAUB $.
+
+    $( 9 + 10 equals 21.  This astonishing thesis lives as a meme on the
+       internet, and may be believed by quite some people.  At least repeated
+       requests to falsify it are a permanent part of the story.  Prof.  Loof
+       Lirpa did not rest until he finally came up with a computer verifiable
+       mathematical proof, that only a fool can think so.  (Contributed by
+       Prof.  Loof Lirpa, 26-Aug-2023.)  (Proof modification is discouraged.)
+       (New usage is discouraged.) $)
+    9p10ne21stupid $p |- ( ( 9 + ; 1 0 ) = ; 2 1 -> F (/) ( 0 x. 1 ) ) $=
```

Comment:
```suggestion
    9p10ne21fool $p |- ( ( 9 + ; 1 0 ) = ; 2 1 -> F (/) ( 0 x. 1 ) ) $=
```
I'd rather not have derogatory terms in set.mm. ("fool" can be alllowed because of the "April Fools" phrase)